oci/metrics-proxy/image.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1
+upload:
+  - source: canonical/metrics-proxy-rock
+    commit: 66763a5703df2e3175b87c0da69b29103da47169
+    directory: "0.1.1"
```

Comment:
Is it worth adding the [0.1.0](https://github.com/canonical/metrics-proxy-rock/tree/main/0.1.0) rock as well?